src/client/process.ts

```diff
@@ -134,22 +137,13 @@ export class PowerShellProcess {
         return this.sessionDetails
     }
 
-    public showConsole(preserveFocus: boolean) {
-        if (this.consoleTerminal) {
-            this.consoleTerminal.show(preserveFocus);
-        }
-    }
-
-    public async eval(line: string) {
-        if (this.consoleTerminal) {
-            this.consoleTerminal.sendText(line)
-        }
-    }
 
     public async scrollToBottom() {
-        this.consoleTerminal.show(false)
-        await utils.sleep(100)
-        await vscode.workspace.nvim.command("wincmd w")
+        if (this.config.integratedConsole.showOnStartup) {
```

Comment:
what if user set showOnStartup: false but then manually show the terminal?

Reply:
Good call. This was naive. Any idea if there’s a way to test if a buffer/terminal is visible?